src/core/components/stats.js

```diff
@@ -49,17 +49,17 @@ module.exports = function stats (self) {
     })
 
     if (opts.poll) {
-      human(opts.interval || '1s', (err, value) => {
-        if (err) {
-          return stream.end(errCode(err, 'ERR_INVALID_POLL_INTERVAL'))
-        }
+      try {
+        const value = human(opts.interval || '1s')
 
         interval = setInterval(() => {
           bandwidthStats(self, opts)
             .then((stats) => stream.push(stats))
             .catch((err) => stream.end(err))
```

Comment:
This is not related to your changes, but maybe worth fixing: if we end the stream here should we also `clearInterval()`?

Reply:
Good point! I will do that